### PR TITLE
feat(max zoom): add option to select exported geopackages max zoom

### PR DESCRIPTION
**Pull request by shimoncohen**

### CODE REVIEW

File: src/exporter/components/export/export-dialog.tsx [hunk 1]
```diff
@@ -138,10 +142,14 @@ export const ExportDialog: React.FC<ExportDialogProps> = observer((props) => {
 
   useEffect(() => {
     if (exporterStore.hasError(ExportStoreError.DUPLICATE_PATH)) {
-      setServerErrors({ duplicate: 'export.dialog.duplicate-path.text' });
+      setServerErrors({ ...serverErrors, duplicate: 'export.dialog.duplicate-path.text' });
       exporterStore.cleanError(ExportStoreError.DUPLICATE_PATH);
     }
-  }, [exporterStore, exporterStore.errors]);
+    else if(exporterStore.hasError(ExportStoreError.BBOX_TOO_SMALL_FOR_RESOLUTION)) {
```
Comment (asafMasa) [inline src/exporter/components/export/export-dialog.tsx:148]: is there a test for multiple errors? 
for the "setServerErrors" and clean specific error while other remains?

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍 nice job

File: src/exporter/components/export/export-dialog.tsx (hunk 1 above)
Comment (shimoncohen) [reply src/exporter/components/export/export-dialog.tsx:148]: Added test

File: src/exporter/components/export/export-dialog.tsx [hunk 2]
```diff
@@ -265,11 +273,15 @@ export const ExportDialog: React.FC<ExportDialogProps> = observer((props) => {
                 null
             }
             {
-              (serverErrors.duplicate) ?
+              Object.entries(serverErrors).map(([error, value]) => {
+                console.log(`Error: ${error}, Value: ${value}`);
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply src/exporter/components/export/export-dialog.tsx]: Removed

File: src/common/i18n/en.json [hunk 3]
```diff
@@ -30,8 +30,8 @@
   "export.dialog-field.bottom_left_lon.label": "Bottom Left Lon",
   "export.dialog-field.directory_name.placeholder": "Directory Name (English Only)",
   "export.dialog-field.package_name.placeholder": "Package Name (English Only)",
-  "export.dialog-field.min_zoom.label": "Min Zoom",
-  "export.dialog-field.max_zoom.label": "Max Zoom",
+  "export.dialog-field.min_zoom.label": "Min Zoom Level",
```
Comment (asafMasa) [inline src/common/i18n/en.json:33]: update the RTL file

**Review by asafMasa — APPROVED** [review]

**Pull request merged**